.circleci/config.yml

```diff
@@ -61,6 +61,47 @@ jobs:
           command: |
             cd build
             make install
+  build_gpu:
+    machine:
+      resource_class: gpu.nvidia.small
+      image: ubuntu-1604-cuda-10.1:201909-23
+    steps:
+      - checkout
+      - run:
+          name: Install apt packages
+          command: |
+            sudo apt-get update
+            sudo apt-get install -y git-core build-essential cmake
+      - run:
+          name: Initialize submodules
+          command: |
+            git submodule init
+            git submodule update
+      - run:
+          name: Build
+          command: |
+            mkdir build
+            cd build
+            cmake ../ \
+              -DCMAKE_C_FLAGS="-Werror -Wno-deprecated-declarations" \
+              -DCMAKE_CXX_FLAGS="-Werror -Wno-deprecated-declarations" \
+              -DTP_ENABLE_SHM=OFF \
+              -DTP_ENABLE_CMA=OFF \
+              -DTP_USE_CUDA=ON \
+              -DTP_ENABLE_CUDA_IPC=ON \
+              -DTP_ENABLE_IBV=OFF \
+              -DTP_BUILD_TESTING=ON
```

Comment:
Are we disabling all the other ones because they are failing, or because you want this to be focused on CUDA only?

Reply:
A bit of both actually, the other ones except for SHM were disabled on the cpu Linux builds, and shm was failing here.